content/blog/2022-01-31-january-22-community-gems.md

```diff
@@ -0,0 +1,191 @@
+
+### [What is the difference between Kubeflow pipelines and DVC pipelines?](https://discord.com/channels/485586884165107732/563406153334128681/922728960478035978)
+
+This is a fantastic question! Thanks for asking @ramakrishnamamidi!
+
+A major difference is that DVC focuses primarily on ML _development_ and adding
+lightweight functionality on top of existing projects, which might be reusable
+in deployment in some cases.
+
+Kubeflow focuses on _deployment_ and building on top of Kuberenetes, which could
+be used during development but requires more up-front effort.
+
+### [Could DVC be a good alternative to LFS for game development?](https://discord.com/channels/485586884165107732/485586884165107734/928336349487067196)
+
+Thanks for such an interesting question @CB!
+
+Yes! We have community members that use DVC to handle their large files in game
+development.
+
+There are several other use cases we've seen for DVC outside of machine learning
+and data science. Some users have used DVC to track build artifacts for
+deployment systems and to track performance data alongside design iterations and
+simulation tools.
+
+### [Does DVC run on JSON/YAML configuration files for all things?](https://discord.com/channels/485586884165107732/563406153334128681/928779586622332938)
+
+This is a great question about large projects with a lot of dependencies from
+@SolemnSimulacrum!
+
+All of the dependencies you list in `dvc run` are in fact configured in the
+`dvc.yaml` file. `dvc run` is a convenience for adding a pipeline stage to this
+file and then doing `dvc repro` on that stage. It's completely acceptable and
+even encouraged to directly edit `dvc.yaml` if that's easier.
+
+For example, if you are currently executing a command like this:
+
+```dvc
+dvc run -f -n prune \
+        -d ./DepFiles_0/ \
+        -d ./DepFiles_1/ \
+        -d ./DepFiles_2/ \
+        -d ./src/.py \
+        -d ./packages/.py \
+        -d ./scripts/.py \
+        -d ./data/.npy \
+```
+
+You could add those directly to the `dvc.yaml` like this:
+
+```yaml
+stages:
+  train:
+    cmd: python recommender_sys.py
+    deps:
+      - ./DepFiles_0/
+      - ./DepFiles_1/
+      - ./DepFiles_2/
+      - ./src/.py
+      - ./packages/.py
+      - ./scripts/.py
+      - ./data/.npy
+    params:
+      - lr
+      - momentum
+      - algorithm
+      - num_classes
+      - batch_size
+      - num_epochs
+    outs:
+      - model.pt
+    metrics:
+      - results.json:
+          cache: false
+    plots:
+      - predictions.json:
+          cache: false
+```
+
+### [I'm setting up MLOps at my company from scratch and we use GitLab and Cloudera DS workbench. What are the best resources to get started with CML?](https://discord.com/channels/485586884165107732/728693131557732403/923785806848614461)
+
+This is a great question from @dvc!
+
+We recommend you start with the CML docs website: https://cml.dev/
+
+You can find some tutorials on our blog: https://dvc.org/blog
```

Comment:
Is there a reason for the links to be in this format and not [CML docs website](https://cml.dev/)?